client/state/selectors/is-gutenberg-opt-out-enabled.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import getSelectedEditor from 'state/selectors/get-selected-editor';
+import isClassicEditorForced from 'state/selectors/is-classic-editor-forced';
+
+export const isGutenbergOptOutEnabled = ( state, siteId ) => {
+	return (
+		get( state, [ 'gutenbergOptIn', siteId ], true ) &&
+		getSelectedEditor( state, siteId ) === 'gutenberg-iframe' &&
```

Comment:
Any reason we want to use gutenberg-iframe here as opposed to `!== 'classic'` (i.e. so that we cover the different gutenberg cases?)

Reply:
good point  - have fixed this.